client/signup/wpcom-login-form/index.jsx

```diff
@@ -1,13 +1,13 @@
 /**
  * External dependencies
  */
-var React = require( 'react' ),
-	debug = require( 'debug' )( 'calypso:signup:wpcom-login' );
+import React from 'react';
+const debug = require( 'debug' )( 'calypso:signup:wpcom-login' );
```

Comment:
👍 that's good to know. (Since this is part of a bulk change, I was wary of introducing an intermediary variable in case of collisions.)

I think using the shorter syntax has two potential downsides - you've already mentioned the first (it deviates from an established pattern in the codebase). The second is that Webpack 2 won't be able to tree shake this import due to the CommonJS notation. However, I highly doubt we'd reach a situation where we'd need to shake `debug` off during a build.